src/sayt/index.ts

```diff
@@ -11,7 +11,8 @@ class Sayt {
     highlight: (value, replacement) => {
       const query = this.flux.store.getState().data.autocomplete.query;
       return value.replace(new RegExp(escapeRegexp(query), 'i'), replacement);
-    }
+    },
+    setInactive: () => this.setInactive()
```

Comment:
no need to expose this function directly as SAYT is already listening for `sayt:hide`